bin/run-cloud

```diff
@@ -0,0 +1,15 @@
+ #! /usr/bin/env bash
+
+# DOC: Run deploy scripts from Docker.
+
+# Usage: 
+# bin/build-cloud
+# bin/run-cloud <command> <tag> <config_file>
+
+export COMMAND="${1:-deploy}"
+export IMAGE_TAG="${2:-latest}"
+export CONFIG="${3:-cloud/deploys/dev_aws/civiform_config.sh}"
+export AWS_ACCESS_KEY_ID=$(aws --profile default configure get aws_access_key_id)
+export AWS_SECRET_ACCESS_KEY=$(aws --profile default configure get aws_secret_access_key)
+echo "Running ${COMMAND} ${IMAGE_TAG} ${CONFIG}"
+docker compose -f cloud/docker-compose.yaml up
```

Comment:
Will deploy scripts need to check out civiform repo at all? Or they can just run `docker run` and use the cloud image?

Reply:
It will not need to checkout main repo, docker-compose file and scripts to run it will be in civiform-deploy repo itself. 
It's convenient to have it in the main repo for testing for now.

Reply:
Add a TODO for that?

Reply:
I don't think todo is necessary here. That's a separate task I'm planning to do.